tools/diff-processor/rules/rules_field_test.go

```diff
@@ -560,6 +560,81 @@ var fieldRule_ShrinkingMaxTestCases = []fieldTestCase{
 	},
 }
 
+func TestFieldRule_AddingSubfieldToConfigModeAttr(t *testing.T) {
+	for _, tc := range fieldRule_AddingSubfieldToConfigModeAttrTestCases {
+		tc.check(fieldRule_AddingSubfieldToConfigModeAttr, t)
+	}
+}
+
+var fieldRule_AddingSubfieldToConfigModeAttrTestCases = []fieldTestCase{
```

Comment:
Nit: Perhaps worth adding another test or two, like removing a subfield. It would help define the edges of this behavior.